extensions/oplogPopulator/modules/Connector.js

```diff
@@ -6,6 +6,7 @@ const connectorParams = joi.object({
     name: joi.string().required(),
     config: joi.object().required(),
     buckets: joi.array().required(),
+    isRunning: joi.boolean().required(),
```

Comment:
`isRunning` is good enough for now, as long as the allocator is always give a _static_ list of connectors.

however, we should (eventually, not here) implement a mecanism to create more connectors when we reach a limit (max mongo query size, pre-configured number of collections...) : and at that point the allocator should be able to allocate more connectors if needed. But this is not for today.